js/src/components/google-account-card/request-full-access-google-account-card.js

```diff
@@ -21,7 +21,7 @@ import './request-full-access-google-account-card.scss';
  * @param {string} props.additionalScopeEmail Specify the email to be requested additional permission scopes to Google.
  * @fires gla_google_account_connect_button_click with `{ action: 'scope', context: 'reconnect' }`
  * @fires gla_google_account_connect_button_click with `{ action: 'scope', context: 'setup-mc' }`
- * @fires gla_documentation_link_click with `{ context: 'setup-mc-accounts', link_id: 'required-google-permissions', href: 'https://woocommerce.com/document/google-listings-and-ads/#required-google-permissions' }`
+ * @fires gla_documentation_link_click with `{ context: 'setup-mc-accounts', link_id: 'required-google-permissions', href: 'https://woocommerce.com/document/google-for-woocommerce/get-started/requirements/#general-requirements' }`
```

Comment:
Are we sending them to the right link here when they haven't accessed all scopes?

What about using https://woocommerce.com/document/google-for-woocommerce/get-started/setup-and-configuration/#required-google-permissions

Reply:
I agree. Changed here https://github.com/woocommerce/google-listings-and-ads/pull/2486/commits/679db1015b964931a28bfe85b562d25390f9e8a9